torch/_meta_registrations.py

```diff
@@ -1166,6 +1167,10 @@ def meta_binop_inplace_alpha(self, other, alpha=1):
     return self
 
 
+@register_meta([aten.round.default, aten.round.decimals])
+def meta_round(self, **kwargs):
+    return _elementwise_meta(self, type_promotion=ELEMENTWISE_PRIM_TYPE_PROMOTION_KIND.DEFAULT)
```

Comment:
@mruberry @ezyang is it ok to reuse this function that's used to find the correct meta information (strides, dtype). It seems like the meta and prim implementation for all elementwise functions should be the same?

Reply:
> Note that if you wanted to write a meta function for an operation like refs.add then the meta function for prims.add would not be correct.

how come?

Reply:
The prims don't support features like type promotion, and `torch.add`/`refs.add` in particular has unique behavior for handling its `alpha` parameter